readme.md

```diff
@@ -34,13 +34,18 @@ unicorns
 
 Get `stdin` as a string.
 
-Returns a promise.
+Returns a promise that is resolved when the 'end' event fires on the `stdin` stream, indicating that there is no more data to be read. 
+
+In a TTY context, a promise that resolves to empty string is returned. 
```

Comment:
As I read the code, the proposed docs are corrected here in that an empty string is returned in a TTY context:

```javascript
if (stdin.isTTY) {
			resolve(ret);
			return;
}
```